tests/test_fast_scan.py

```diff
@@ -33,6 +33,38 @@ def test_PQ4_accuracy(self):
         assert recall_at_1 > 0.6
         # print(f'recall@1 = {recall_at_1:.3f}')
 
+    def test_PQ4_speed(self):
+        # only test while building with avx2
+        print(faiss.has_AVX2)
+        if not faiss.has_AVX2:
+            return
```

Comment:
Please use the `@unittest.skipIf()` decorator instead.

Reply:
Thanks! I do not even know this usage.